client/components/plans/plan-actions/index.jsx

```diff
@@ -194,9 +194,10 @@ module.exports = React.createClass( {
 	},
 
 	managePlanButton: function() {
+		var link = "/purchases/" + this.props.site.domain + "/" + this.props.siteSpecificPlansDetails.id;
```

Comment:
The linter wants these to be either single quotes or grave accents (template strings).